internal/mq/mqimpl/rocksmq/server/rocksmq_impl.go

```diff
@@ -166,7 +167,7 @@ func NewRocksMQ(name string, idAllocator allocator.Interface) (*rocksmq, error)
 	// L0:No Compression
 	// L1,L2: ZSTD
 	optsKV.SetNumLevels(3)
-	optsKV.SetCompressionPerLevel([]gorocksdb.CompressionType{0, 7, 7})
+	optsKV.SetCompressionPerLevel([]gorocksdb.CompressionType{0, 1, 7})
```

Comment:
don't play this trick.
this require too much dependency

Reply:
and we should let this to be configurable

Reply:
> don't play this trick. this require too much dependency

Type 1 means snappy, was default compression type, will not require new dependency. And I will make this configurable.